include/rdata.h

```diff
@@ -1,26 +1,23 @@
 #ifndef _MY_RDATA
 #define _MY_RDATA
 #include <include/udata.h>
-class UserData;
+
 class Model;
```

Comment:
stray class Model? was removed in udata.h

Reply:
We need it for `    ReturnData(const UserData *udata, const Model *model);
`